src/native/corehost/hostmisc/pal.windows.cpp

```diff
@@ -52,6 +52,8 @@ pal::string_t pal::get_timestamp()
     std::time_t t = std::time(0);
     const std::size_t elems = 100;
     char_t buf[elems];
+
+#pragma warning(suppress : 4996)  // error C4996: 'gmtime': This function or variable may be unsafe.
```

Comment:
Does this disable the warning for the rest of the file?

Reply:
Nope. `suppress` pops after being applied to the next line.